src/DefaultResolverApp.php

```diff
@@ -27,12 +24,6 @@ class DefaultResolverApp implements ResolverApp {
 
 	public function __construct() {
 		$this->routeMap = new RouteMap();
-
-		// configuration annotation reading for api model validation
-		AnnotationRegistry::registerAutoloadNamespace(
```

Comment:
We're sure we're good to remove this? I forget - was this duplicated by bootstrap.php calls when the resource is known to use this?

Reply:
Yes, this is one of the breaking changes we discussed. It will be required now in the bootstrap of projects using it. This way no longer is supported in symfony 3.4+